pkg/collector/check/jmx.go

```diff
@@ -35,7 +35,15 @@ func IsJMXInstance(name string, instance integration.Data, initConfig integratio
 		return false
 	}
 
-	x, ok := rawInstance["is_jmx"]
+	x, ok := rawInstance["loader"]
```

Comment:
Note: `loader` will take precedence over `is_jmx` at both init and instance config level.

Reply:
I think this is fine, and makes sense, but should be documented.

Reply:
Sure, will handle that in a separate PR.